third-party/Dockerfile-grimoirelab-nomos

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2016-2017 Bitergia
+# GPLv3 License
+#
+# Build GrimoireLab packages, install them, check them
+#
+
+FROM grimoirelab/full
+MAINTAINER Jesus M. Gonzalez-Barahona <jgb@bitergia.com>
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+ADD build/fossology-common_3.6.0-1_amd64.deb /tmp
+ADD build/fossology-nomos_3.6.0-1_amd64.deb /tmp
+# install dependencies
+RUN sudo apt-get update && \
+    sudo apt-get -y install /tmp/fossology-common_3.6.0-1_amd64.deb \
+    	/tmp/fossology-nomos_3.6.0-1_amd64.deb \
+        && \
+    sudo apt-get clean && \
+    sudo find /var/lib/apt/lists -type f -delete && \
+    sudo rm /tmp/fossology-common_3.6.0-1_amd64.deb \
+       /tmp/fossology-nomos_3.6.0-1_amd64.deb
+
```

Comment:
Nomos needs to be set up by running the conf script `pgsql-conf-fix.sh` located in `/usr/share/fossology/setup`.

We could add also other packages needed by Graal, a possible modification could be:
```
@@ -15,12 +15,25 @@ ADD build/fossology-nomos_3.6.0-1_amd64.deb /tmp
 RUN sudo apt-get update && \
     sudo apt-get -y install /tmp/fossology-common_3.6.0-1_amd64.deb \
        /tmp/fossology-nomos_3.6.0-1_amd64.deb \
+        python3 \
+        python3-pip \
+        cloc \
         && \
     sudo apt-get clean && \
     sudo find /var/lib/apt/lists -type f -delete && \
     sudo rm /tmp/fossology-common_3.6.0-1_amd64.deb \
        /tmp/fossology-nomos_3.6.0-1_amd64.deb
 
+RUN pip3 install setuptools
+RUN pip3 install lizard>=1.16.3
+RUN pip3 install pylint>=1.8.4
+RUN pip3 install flake8>=3.7.7
+RUN pip3 install networkx>=2.1
+RUN pip3 install pydot>=1.2.4
+RUN pip3 install bandit>=1.4.0
+
+CMD /usr/share/fossology/setup/pgsql-conf-fix.sh <--- setup command needed by Nomos
+
 # Entrypoint
 ENTRYPOINT [ "/entrypoint.sh" ]
```

Reply:
I think Python3 is already in the container (it is used to run GrimoireLab tools). Same for setuptools, I think. Why are you proposing several different `pip install` commands? Can't we just install the main ones, and the rest would come in as dependencies? Some of them seem to be related to testing, not to the running packages (eg `pylint` or `flake8`): do we really need them in the container?

Reply:
> I think Python3 is already in the container (it is used to run GrimoireLab tools). Same for setuptools, I think.

OK

> Why are you proposing several different pip install commands? Can't we just install the main ones, and the rest would come in as dependencies?

Yes, you are right, it's better to get them as dependencies of graal.

> Some of them seem to be related to testing, not to the running packages (eg pylint or flake8): do we really need them in the container?

Pylint and flake8 are used by some graal analyzers (https://github.com/chaoss/grimoirelab-graal/tree/master/graal/backends/core/analyzers).

Reply:
OK. Then, can I understand that from your diff, only the following two lines are needed?:

```
+        cloc \
...
+CMD /usr/share/fossology/setup/pgsql-conf-fix.sh 
```

For the second one, are we really needing it, even when we're using MySQL and not PostgreSQL?

Reply:
I tried the PR again, from a fresh env. I included only cloc and eveything worked smoothly.

It would be possible to change the name of the image with something not related to nomos (maybe in the future there will be more things inside), for instance `full-thirdparties` WDYT?